learn-python/debugging/Python_Workshop_cells.ipynb

```diff
@@ -0,0 +1,1309 @@
+{
```

Comment:
Do you want to commit this jupyter notebook with the data loaded into it? As you already provide the data I think it might be better to clear all outputs of the notebook and save it again, to reduce its size.

Reply:
can you do that @paulboehnke ?